opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
How about `"rpc.target.service"`, `"rpc.target.hostname"`, etc? (and the peer can fill in `"rpc.source.hostname"` and `"...ipv[46]"`, etc, albeit without a `service`)